x/distribution/types/delegator_info.go

```diff
@@ -0,0 +1,55 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// distribution info for a delegation
+type DelegatorDistInfo struct {
```

Comment:
"distribution info" isn't a terribly informative name - `type DelegatorDistributionWithdrawalInfo` maybe?

Reply:
ahhhh - I like it as is (Jae's suggested name) - I think `DelegatorDistributionWithdrawalInfo` is too verbose - after all this _is_ the distribution information independent of if the distributed coins are withdrawn or not - I think it's fine, I'll spoof up the comment though